content/_index.md

```diff
@@ -12,8 +11,8 @@ codes:
       ak.startup('localhost', 5555)
 
       # Generate two large arrays
-      a = ak.randint(0,2**32,2**10)
-      b = ak.randint(0,2**32,2**10)
+      a = ak.randint(0,2**32,2**38) # ----> Won't fit on a single machine!
+      b = ak.randint(0,2**32,2**38) # /     1TB of random integers.
```

Comment:
Can we space this out so that the comments "stay" in the same place when switching the tabs between Arkouda and Numpy?
Right now these comments jump around due to the code changes when switching between the two tabs.